src/utils/timeutils.ts

```diff
@@ -1,7 +1,4 @@
-import moment, { Moment } from "moment";
-import "moment/dist/locale/nb";
```

Comment:
Have you tested that this still displays the "x mins ago" in Norwegian and not English?

Reply:
To clarify, this line, and the other "locale" line underneath are not there for time zone handling, but to get time strings in Norwegian formats